### ISSUE TITLE
Using @DocumentationExample to define two entries in a Map

### ISSUE THREAD

**Issue by fdtaillefer** [stoicflame/enunciate#919]
The @DocumentationExample annotation doesn't seem to allow overriding both default entries for a Map. From reading #720, I am under the impression that it's meant to be possible.

For the example's sake, I am working with a Map<String, String> named description. The default documentation output, with no DocumentationExample, offers two entries and looks like this:
```
"description" : {
    "property1" : "...",
    "property2" : "..."
 }
```
**Case 1**
If i use  the annotation with one key-value entry (`@DocumentationExample("key val")`), or the javadoc tag (`@documentationExample key val`), the output uses the key once but the value twice, coming out like this:
```
"description" : {
    "key" : "val",
    "property2" : "val"
 }
```
**Case 2**
If i use  the annotation with two key-value entries (`@DocumentationExample(value = "key1 val1", value2 = "key2 val2")`), the output is the same but using the second key:
```
 "description" : {
    "key2" : "val1",
    "property2" : "val1"
 }
```
Am I using it wrong? Is there a way to define both key-value pairs in the output?

**Comment by stoicflame** [comment]
I think I fixed it at 0b1c77a

**Comment by stoicflame** [comment]
[Enunciate 2.12](https://github.com/stoicflame/enunciate/releases/tag/v2.12.0) has been released.
